README.md

```diff
@@ -54,10 +54,10 @@ python makeSkimInput.py -r input/dict_skim_data_2018.py -w input/input_sets_skim
 ```
 Note: this script uses the python file lists in [TreeMaker/Production/python](https://github.com/TreeMaker/TreeMaker/tree/Run2/Production/python) to determine the number of files to chain together for each sample.
 
-To submit jobs to Condor (add the flag `-k` to reuse the existing CMSSW tarball):
+To submit jobs to Condor (add the flag `-k` to reuse the existing CMSSW tarball or `-d` to use in DRYRUN mode):
 ```
 cd batch
-./SKsub.sh -t MC,Signal,Data -y 2016,2017,2018
+./SKsub.sh -t MC,Signal,SignalScan,Data -y 2016,2017,2018
```

Comment:
I would put SignalScan in the next command (with Fast) since the note below remarks that the Fast skim should only run after the scanning step (which is also true for SignalScan)

Reply:
changed